### ISSUE TITLE
Requested file format unknown: escn

### ISSUE THREAD

**Issue by hedgehog90** [godotengine/godot#29327]
In the latest build I get an alert prompt every time I test a scene that inherits an escn or if I test the escn itself.
Has the escn extension been deprecated or is this a bug?

**Comment by qarmin** [comment]
I think that escn format is deprecated, and you should use tscn(text format) or scn(compresed scene format) instead.
This is a popup with supported types of saved scenes(Godot master)
![Zrzut ekranu z 2019-05-31 06-21-54](https://user-images.githubusercontent.com/41945903/58681417-899d3a00-836c-11e9-929c-4500742aeead.png)

**Comment by akien-mga** [comment]
`escn` is not deprecated, no. It should be imported to `scn` automatically.

**Comment by Zireael07** [comment]
I used escn in 3.1.1. (Godot Blender export output) and had the same alert pop up.

**Comment by hedgehog90** [comment]
I ask because the official Blender Godot Exporter (which I'm helping to improve) exports escn and it seems to have been overlooked in Godot by recent changes.

**Comment by Obr00007576** [comment]
> 我问是因为官方的Blender Godot出口商（我正在帮助改进）出口escn，并且在Godot中，最近的变化似乎忽略了它。

Yeah, it seems like that. But if you want to modify the imported 3d models or nodes(in godot) there is no use to modify the origin exported file from blender. Dragging the escn file exported from blender straightly the the current node will result in unmodifiability of the nodes. You need to right click the parent node where you want to add the node and  select 'Merge From Scene' to choose the exported escn file. Then, you can choose nodes in the file and those new nodes will be modifiable. You can also create an empty scene and just do as what I said, then just save the scene with another name, the new scene will be alright, you can save it and modify it and it's absolutely same as the file you exported from blender. It is somehow troublesome, but is the best to solve the problem with such bug.

**Comment by Maxattax97** [comment]
I had this issue when importing a project which had `.import` files ignored, see here: https://github.com/github/gitignore/pull/2827

Removing the line in `.gitignore` fixed it, but for those that wander onto this comment: be wary there are security concerns (at least as of writing) -- read the PR for details.
